src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="Xamarin.Android.Tools.BootstrapTasks.GenerateProfile" AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" />
-  <Target Name="_Foo" BeforeTargets="CoreCompile">
+  <ItemGroup>
+    <SharedRuntimeBuildPath Include="..\..\bin\$(Configuration)\lib\xbuild-frameworks\MonoAndroid\" />
+    <_SharedRuntimeAssemblies Include="@(SharedRuntimeBuildPath->'%(Identity)\\v1.0\*.dll');@(SharedRuntimeBuildPath->'%(Identity)\\v6.0\*.dll')"/>
+  </ItemGroup>
+
+  <Target Name="_Foo" BeforeTargets="CoreCompile"
+	  Inputs="@(_SharedRuntimeAssemblies)"
+	  Outputs="$(_GeneratedProfileClass)"
```

Comment:
*How's this work*, given that `$(_GeneratedProfileClass)` is a property declared *within this target*, 3 lines down?

I mean, if that *does* work, that's kinda cool, but I didn't think that this was possible/supported in xbuild/MSBuild...